src/codegen.rs

```diff
@@ -4042,4 +4144,107 @@ int main() {{
                 .expect("Could not convert stderr to UTF8")
         );
     }
+
+    #[test]
+    #[should_panic(expected = "unfinished test")]
```

Comment:
Sorry for flip-flopping on this, but could you actually change this (and the other unimplemented tests) to
```suggestion
    #[ignore = "unfinished test"]
```
It will help me keep track of tests that need to be changed in the long-term. The #should_panic strategy is good for the short term, while you're working on a PR, but I don't want to check in code that hides unfinished tests without calling them out in some way. Using #ignore will at least make them show up as ignored when I run the test suite.